code/obj/machinery/nuclearbomb.dm

```diff
@@ -284,6 +286,16 @@
 		return timeleft
 
 	proc/take_damage(var/amount)
+		if(!isitspacemas)
+			switch(src.health)
+				if(80 to 125)
+					src.icon_state = "nuclearbomb1"
+				if(40 to 79)
+					if(!isitspacemas)
+						src.icon_state = "nuclearbomb2"
+				if(1 to 39)
+					if(!isitspacemas)
```

Comment:
These inner ifs are redundant. Also the ranges technically don't cover all values (e.g. taking damage from 125 to 39.5 wouldn't update the sprite as 39.5 doesn't lie in any of the ranges). But I can just fix that after merging it or something.